content/S2015/S15-event-googletalk.md

```diff
@@ -1,7 +1,7 @@
 Title: Google Talk
 Date: 2015-05-14 18:30
 Category: Events
-Slug: google-talk
+Slug: google-talk, talks
```

Comment:
So this line is actually for a Slug, not tags. There should only be one slug (it's what will come up in the URL after wics.uwaterloo.ca/ but you might know that already :P). You need to add a new line for Tags looks like this file doesnt have any